### PR TITLE
feat(ui): horizontal pagination added in `allRecentJobs`

### PR DESCRIPTION
**Pull request by soham4abc**
## Description
Horizontal style pagination added in the `/allRecentJobs` page.

### Changes
New packages have been used in this PR. Be sure to run `yarn install` when starting the project

## How to test
Run `yarn install` and start the server.
Visit the `/allRecentJobs` and scroll down to see the pagination.

## Video Demo

https://user-images.githubusercontent.com/63705023/175760990-fa73245d-14f2-4ace-a862-d9f802fa9159.mp4

### CODE REVIEW

**Comment by soham4abc** [comment]
PTAL @GMishx  @shaheemazmalmmd

**Comment by GMishx** [comment]
Personally, I would recommend to move the navigation to center or to right rather than on left.

Also, is it possible to add a text box next to it? Use case being: in case there are say 100 pages, one can directly jump page 42 rather than juggling with multiple mouse clicks.

File: package.json [hunk 1]
```diff
@@ -3,12 +3,15 @@
...
   "dependencies": {
+    "@material-ui/core": "^4.12.4",
+    "@material-ui/lab": "^4.0.0-alpha.61",
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
     "array-to-tree": "^3.3.2",
     "bootstrap": "4.6.0",
     "js-cookie": "^2.2.1",
+    "material-ui": "^0.20.2",
```
Comment (GMishx) [inline package.json:14]: image

However, recommended package is already included. Do we still need it?

Comment (soham4abc) [reply package.json]: Removed the package.. working fine without it

**Comment by soham4abc** [comment]
> Personally, I would recommend to move the navigation to center or to right rather than on left.
> 
> Also, is it possible to add a text box next to it? Use case being: in case there are say 100 pages, one can directly jump page 42 rather than juggling with multiple mouse clicks.


Made something like this:
Screenshot from 2022-06-29 10-53-28

**Review by GMishx — APPROVED** [review]
Changes looks good.

**Pull request merged**